language-support/java/bindings/src/main/java/com/daml/ledger/javaapi/data/codegen/ContractTypeCompanion.java

```diff
@@ -5,12 +5,17 @@
 
 import com.daml.ledger.javaapi.data.Identifier;
 
+import java.util.List;
+
 /** The commonality between {@link ContractCompanion} and {@link InterfaceCompanion}. */
 public abstract class ContractTypeCompanion {
   /** The full template ID of the template or interface that defined this companion. */
   public final Identifier TEMPLATE_ID;
+  //TODO: Need to figure out the right type params here
+  private final List<ChoiceMetadata<?, ?, ?>> choices;
```

Comment:
@S11001001 I was about to ask about this and saw your comment from yesterday. I think adding that `Marker` class will solve my problem

Reply:
We're going to change it again in #15125, so might as well get started now.